app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,24 @@
   <title>Audioscape</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <script src="//cdn.jsdelivr.net/rsvp/3.0.6/rsvp.min.js"></script>
+  <!-- Firebase -->
+  <script src="https://cdn.firebase.com/js/client/2.0.2/firebase.js"></script>
+
+  <!-- GeoFire -->
+  <script src="https://cdn.firebase.com/libs/geofire/3.2.1/geofire.min.js"></script>
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <% if current_user %>
+  <div id='user<%=current_user.id%>' style='display:none'></div>
```

Comment:
If you are going to combine user with id, the convention is to use an underscore `_` or dash `-` between user and the number. 

If the goal of this is *only* to store the user_id somewhere, you'd be much better off holding it in a JS object.  Accessing the DOM is one of the most expensive things you can do in JS.

remember you can write this in .html.erb files:
```ruby
<script type="text/javascript">
  var AUDIOSCAPE = AUDIOSCAPE || {};
  AUDIOSCAPE.user_id = <%= current_user.id %>
</script>